plugins/ltac2/tac2stdlib.ml

```diff
@@ -397,6 +397,14 @@ let () =
     (bool @-> list rewriting @-> clause @-> option (thunk unit) @-> tac unit)
     Tac2tactics.rewrite
 
+let to_helper v = uthaw constr_with_bindings v
+let helper = make_to_repr to_helper
+
+let () =
+  define "tac_setoid_rewrite"
+    (option bool @-> helper @-> occurrences @-> option ident @-> tac unit)
```

Comment:
```suggestion
    (option bool @-> uthaw constr_with_bindings @--> occurrences @-> option ident @-> tac unit)
```
I think this works

In any case I don't like having a toplevel `helper` with such a generic name

You could also do
~~~ocaml
let () =
  let to_helper v = ... in
  let helper = ... in
  define ...
~~~
if you prefer

Reply:
Thanks, I didn't know about `@-->`.